src/commands/download-p5.cjs

```diff
@@ -0,0 +1,55 @@
+/// @ts-check
+const https = require('https');
+const fs  = require('fs')
+const upath = require('upath');
+
+
+const command = "download-p5 [dir]";
+const desc = "Clone or update p5.js git repository";
+
+// exports.builder = {
```

Comment:
Are you planning to add back these parameters? It's preferred to not commit commented out code.